internal/handlers/irc/irc.go

```diff
@@ -13,7 +15,7 @@ and the IRCSettings that were passed into NewClient
 type Client struct {
 	*girc.Client
 	Settings internal.IRCSettings
-	verbose internal.DebugLogger
+	verbose  internal.DebugLogger
```

Comment:
Not sure if you meant to add extra whitespace here :joy:

Reply:
That was on my end with the logging.

Reply:
Yeah, gofmt did that to make it so that the definitions line up haha. Normally this stuff sorta vanishes in PRs, but I don't think Kevin's setup is doing formatting so that's why it didn't show up until this PR.

Reply:
Oh okay! No problem, let's keep it.